### ISSUE TITLE
Need some documentation

### ISSUE THREAD

**Issue by devvratmiglani** [StarrFox/memobj#62]
I know it's too early as I feel its in development. I just want to know if there will be a documentation.

**Comment by StarrFox** [comment]
yep documentation is definitely planned as the project matures, I'm still working on making certain things nice to use though such as memory objects themselves

I could start on documentation for things that probably work change like Process though

**Comment by devvratmiglani** [comment]
Sounds good, looking forward to the docs whenever they’re ready!

**Comment by StarrFox** [comment]
fixed in https://github.com/StarrFox/memobj/commit/cb7a757da3e1a424eef8bfdf6bc1deda7f6ba93c
